commands/numeric/math.js

```diff
@@ -0,0 +1,28 @@
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+const { evaluate } = require("mathjs");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("math")
+    .setDescription("Evaluates mathematical expressions")
+    .addStringOption((option) =>
+      option.setName("expression").setDescription("Expression to be evaluated"),
+    ),
+  async execute(interaction) {
+    const expression = interaction.options.getString("expression");
+
+    try {
+      const result = evaluate(expression);
+      const embed = new EmbedBuilder().addFields(
+        { name: "Expression", value: `\`${expression}\`` },
+        { name: "Result", value: `${result}` },
+      );
+      await interaction.reply({ embeds: [embed] });
+    } catch (error) {
+      const embed = new EmbedBuilder().setTitle(
+        `❌  Input: \`${expression}\` is not a valid expression.`,
```

Comment:
Just a thought but could you try adding a `\t` here and see if it adds a space between the X and Input? It looks a bit cramped in the embed.

Reply:
I should be more specific. Add it between the X and the Input